doc/developer-guides/hld/hld-security.rst

```diff
@@ -310,7 +312,7 @@ Background
 
 The ACRN hypervisor has ultimate access control of all the platform
 memory spaces (see :ref:`memmgt-hld`). Note that on the APL platform,
-`SGX <https://software.intel.com/sgx>`_ and `TME
+`SGX <https://www.intel.com/content/www/us/en/developer/tools/software-guard-extensions/overview.html>`_ and `TME
 <https://software.intel.com/en-us/blogs/2017/12/22/intel-releases-new-technology-specification-for-memory-encryption>`_
```

Comment:
@dbkinder Looks like this TME blog doesn't exist anymore, and the link redirects to an unrelated place. What do you think we should do with this link?

Reply:
Looks like an updated version of that TME blog is here now: https://itpeernetwork.intel.com/memory-encryption/